tests/test_errorcallback.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the error callback"""
+
+import unittest
+import unittest.mock
```

Comment:
Yes, `mock` is a third-party library that got integrated into the std lib only with Python 3.3+. For Python 2 you would need to add it to `tests_require` in `setup.py` (then please add it to `requirements-dev.txt` too).

About `osc2midi`: the package is in the `examples` directory. I either need to find a way to set up the Python path for the tests correctly or maybe I should put this in its own repo anyway. If you run the tests via `tox` (which you should at least once before creating a PR), the package will be installed and the test runs correctly.